source/shell_integration/bash

```diff
@@ -567,7 +567,7 @@ function __iterm2_precmd () {
     fi
 
     # Get the value of the prompt prefix, which will change $?
-    \local iterm2_prompt_prefix_value="$(iterm2_prompt_prefix)"
+    local iterm2_prompt_prefix_value="$(iterm2_prompt_prefix)"
 
```

Comment:
This is intentional. It means "use the real `local`, even if there is an alias with that name"

Reply:
ah. I usually use single quotes.